android/gradle.properties

```diff
@@ -26,4 +26,4 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.54.0
+FLIPPER_VERSION=0.99.0
```

Comment:
I'm not sure of the implications of this. The Flipper dependency is correlated with the version of React Native, but our SDK forces a downstream dependency of `*` for RN itself. I'm ok with this change, but it may or may not cause problems. Really not sure...

Reply:
We were using the newer version of Flipper in the app code path and this was being overridden, so I made them match.